docs/ReverseProxyToTor.md

```diff
@@ -215,6 +215,135 @@ systemctl restart nginx
 
 Now, visiting `mydomain.com` should show your BTCPay Server instance.
 
+
+## Do all this in a Docker container
+
+Ready made docker image 
+
+  https://hub.docker.com/r/cloudgenius/socator
+
+Code behind the image
+
+  https://github.com/beacloudgenius/socator
+
+
+### SocaTor = SOCAT + TOR
+Based on https://github.com/Arno0x/Docker-Socator
+
+It uses socat to listen on a given TCP port (5000 in this example) and to redirect incoming traffic to a Tor hidden service specified through environment variables. It acts as a relay between the standard web and a hidden service on the Tor network. You can optionally restrict the IP addresses that are allowed to connect to this service by specifying an `ALLOWED_RANGE` environment variable and using CIDR notation.
+
+### Usage
+
+Break free from cloud services providers limitations, secure and protect your bitcoin full node, connect that with a BTC Pay server, all behind TOR.
+Selectively expose the BTC Pay Server payment gateway and API to clearnet using socat+tor running on the Internet.
+
+--------------
```

Comment:
remove

Reply:
Sorry I did not understand this comment. Please help.

Reply:
normally we're not using these dividers in the docs but overall it looks good :)